uint16 user_id
uint16 gesture_id
string gesture_name
